_content/tour/ind/constants.article

```diff
@@ -126,55 +124,56 @@ integer constants are assumed once applied.
     )
     fmt.Println(A3, B3, C3)
 
-Output:
+Keluaran:
 
     1 2 3
 
-If you didn’t want to apply a mathematical pattern, you can perform some math and the
-math is reapplied with an increasing value of iota.
+Jika Anda tidak ingin menerapkan pola matematika, Anda dapat melakukan 
+beberapa perhitungan matematika dan perhitungan itu akan diterapkan kembali 
+dengan nilai iota yang meningkat.
 
     const (
-        Ldate= 1 << iota  //  1 : Shift 1 to the left 0.  0000 0001
-        Ltime             //  2 : Shift 1 to the left 1.  0000 0010
-        Lmicroseconds     //  4 : Shift 1 to the left 2.  0000 0100
-        Llongfile         //  8 : Shift 1 to the left 3.  0000 1000
-        Lshortfile        // 16 : Shift 1 to the left 4.  0001 0000
-        LUTC              // 32 : Shift 1 to the left 5.  0010 0000
-    )
+        Ldate= 1 << iota  //  1 : Geser 1 ke kiri 0.  0000 0001
+        Ltime             //  2 : Geser 1 ke kiri 1.  0000 0010
+        Lmicroseconds     //  4 : Geser 1 ke kiri 2.  0000 0100
+        Llongfile         //  8 : Geser 1 ke kiri 3.  0000 1000
+        Lshortfile        // 16 : Geser 1 ke kiri 4.  0001 0000
+        LUTC              // 32 : Geser 1 ke kiri 5.  0010 0000
+)
 
     fmt.Println(Ldate, Ltime, Lmicroseconds, Llongfile, Lshortfile, LUTC)
 
-Output:
+Keluaran:
 
     1 2 4 8 16 32
 
-You can use this feature like the Log package does for setting flags. In this case, bit
-operations are being applied with increasing values of iota to calculate flag values.
+Anda dapat menggunakan fitur ini seperti yang dilakukan oleh package Log untuk mengatur flag. 
+Dalam kasus ini, operasi bit diterapkan dengan meningkatkan nilai iota untuk menghitung nilai flag.
 
-** Notes
+** Catatan
 
-- Constants are not variables.
-- They exist only at compilation.
-- Untyped constants can be implicitly converted where typed constants and variables can't.
-- Think of untyped constants as having a Kind, not a Type.
-- Learn about explicit and implicit conversions.
-- See the power of constants and their use in the standard library.
+- Konstanta bukanlah variable.
+- Konstanta hanya ada pada saat kompilasi.
+- Konstanta yang tidak memiliki jenis dapat diubah secara implisit di mana konstanta yang memiliki jenis dan variabel tidak.
+- Pikirkan tentang konstanta tak berjenis sebagai memiliki Jenis, bukan Tipe.
+- Pelajari tentang konversi eksplisit dan implisit.
+- Lihat kemampuan konstanta dan penggunaannya dalam pustaka standar.
 
 ** Extra Reading
```

Comment:
Missing translation **Extra Reading to **Bacaan Tambahan

Reply:
ack